cmd/ocm/gcp/create-wif-config.go

```diff
@@ -259,16 +285,29 @@ func createWorkloadIdentityConfiguration(
 	client gcp.GcpClient,
 	displayName string,
 	projectId string,
+	federatedProjectId string,
 ) (*cmv1.WifConfig, error) {
+
+	if federatedProjectId == "" {
+		federatedProjectId = projectId
+	}
+
 	projectNum, err := client.ProjectNumberFromId(ctx, projectId)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to get GCP project number from project id")
 	}
 
+	federatedProjectNum, err := client.ProjectNumberFromId(ctx, federatedProjectId)
```

Comment:
pls move the if clause from #527 

if federatedProjectId == "" {
		federatedProjectId = projectId
                 federatedProjectNum = projectNum
}else{
                 federatedProjectNum, err := client.ProjectNumberFromId(ctx, federatedProjectId)
}




	}